test/assayer-test.js

```diff
@@ -1,110 +1,97 @@
-'use strict'
+'use strict';
 
-const support = require('./support')
-const assayer = require('../lib/assayer')
+const support = require('./support');
+const assayer = require('../lib/assayer');
 
 describe('assayer', () => {
-
   describe('test', () => {
-
     it('returns basic info', async () => {
-      const info = await assayer.test({remoteAgent: 'something'})
+      const info = await assayer.test({ remoteAgent: 'something' });
       expect(info).to.eql({
         isDuplicate: false,
         cause: null,
-        geo: null,
```

Comment:
Likewise, a lot of linting in here.  But the only change was removing these checks for the `geo` output.

Reply:
Re linting, I know with robocop you can do a recursive linting from the comand line.  Maybe that is an option: create a linting branch, lint the whole project, and then base the feature on that?

Reply:
I've been linting files as I touch them, thus far... but perhaps it's time!  I'll open a ticket.